homeassistant/components/zabbix/__init__.py

```diff
@@ -85,6 +87,7 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
     password = conf.get(CONF_PASSWORD)
 
     publish_states_host = conf.get(CONF_PUBLISH_STATES_HOST)
+    include_strings = conf.get(CONF_INCLUDE_STRINGS)
```

Comment:
Don't use `dict.get`, `CONF_INCLUDE_STRINGS` has a default value and will always be set.
```suggestion
    include_strings = conf[CONF_INCLUDE_STRINGS]
```